<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayoutAvancement"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>179</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayoutBoutons">
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Exemple_lecteur_musique/img_lecteur/jouer-a-droite-fleche-icone-6822-32.png</normaloff>../../Exemple_lecteur_musique/img_lecteur/jouer-a-droite-fleche-icone-6822-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Exemple_lecteur_musique/img_lecteur/pause-icone-4803-32.png</normaloff>../../Exemple_lecteur_musique/img_lecteur/pause-icone-4803-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Exemple_lecteur_musique/img_lecteur/arreter-icone-7083-32.png</normaloff>../../Exemple_lecteur_musique/img_lecteur/arreter-icone-7083-32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayoutTemps">
     <item>
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayoutVolume">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Exemple_lecteur_musique/img_lecteur/joueur-volume-icone-9860-32.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayoutVideo"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
